source/platforms/bitbucket_server/_tests/__snapshots__/_bitbucket_server_git.test.ts.snap

```diff
@@ -28,6 +28,6 @@ exports[`the dangerfile gitDSL - BitBucket Server should include \`after\` text
 
 exports[`the dangerfile gitDSL - BitBucket Server should include \`before\` text content of the file 1`] = `"{\\"path\\":\\".gitignore\\",\\"repoSlug\\":\\"projects/PROJ/repos/repo\\",\\"ref\\":\\"d6725486c38d46a33e76f622cf24b9a388c8d13d\\"}"`;
 
-exports[`the dangerfile gitDSL - BitBucket Server should include \`removed\` text content of the file 1`] = `"node_modules*.log/test.py/.vscode.DS_Storecoverage.idea__pycache__/*.pyc*.swp"`;
+exports[`the dangerfile gitDSL - BitBucket Server should include \`removed\` text content of the file 1`] = `""`;
```

Comment:
Is this worth looking into?

Reply:
Yeah already checked it. 
Previously the logic was not correct. 
```
[{"destination":{"components":[".gitignore"],"extension":"gitignore","name":".gitignore","parent":"","toString":".gitignore"},"hunks":[{"destinationLine":1,"destinationSpan":11,"segments":[{"lines":[{"destination":1,"line":"node_modules","source":1,"truncated":false},{"destination":2,"line":"*.log","source":2,"truncated":false},{"destination":3,"line":"/test.py","source":3,"truncated":false},{"destination":4,"line":"/.vscode","source":4,"truncated":false},{"destination":5,"line":".DS_Store","source":5,"truncated":false},{"destination":6,"line":"coverage","source":6,"truncated":false},{"destination":7,"line":".idea","source":7,"truncated":false},{"destination":8,"line":"__pycache__/","source":8,"truncated":false},{"destination":9,"line":"*.pyc","source":9,"truncated":false}],"truncated":false,"type":"CONTEXT"},{"lines":[{"destination":10,"line":"","source":10,"truncated":false}],"truncated":false,"type":"ADDED"},{"lines":[{"destination":11,"line":"*.swp","source":10,"truncated":false}],"truncated":false,"type":"CONTEXT"}],"sourceLine":1,"sourceSpan":10,"truncated":false}],"source":{"components":[".gitignore"],"extension":"gitignore","name":".gitignore","parent":"","toString":".gitignore"},"truncated":false},{"destination":{"components":["banana"],"name":"banana","parent":"","toString":"banana"},"hunks":[{"destinationLine":1,"destinationSpan":1,"segments":[{"lines":[{"destination":1,"line":"bar","source":0,"truncated":false}],"truncated":false,"type":"ADDED"}],"sourceLine":0,"sourceSpan":0,"truncated":false}],"source":null,"truncated":false},{"destination":null,"hunks":[{"destinationLine":0,"destinationSpan":0,"segments":[{"lines":[{"destination":0,"line":"\"use strict\";","source":1,"truncated":false},{"destination":0,"line":"","source":2,"truncated":false},{"destination":0,"line":"module.exports = {","source":3,"truncated":false},{"destination":0,"line":"  runner: \"jest-runner-eslint\",","source":4,"truncated":false},{"destination":0,"line":"  displayName: \"lint\",","source":5,"truncated":false},{"destination":0,"line":"  testMatch: [\"<rootDir>/**/*.js\"],","source":6,"truncated":false},{"destination":0,"line":"  testPathIgnorePatterns: [\"node_modules/\"]","source":7,"truncated":false},{"destination":0,"line":"};","source":8,"truncated":false}],"truncated":false,"type":"REMOVED"}],"sourceLine":1,"sourceSpan":8,"truncated":false}],"source":{"components":["jest.eslint.config.js"],"extension":"js","name":"jest.eslint.config.js","parent":"","toString":"jest.eslint.config.js"},"truncated":false}]
```

This is diff as bitbucket json . If you see there is no `removed`  hunks for `.gitignore` file.